css/stylesheet.css

```diff
@@ -89,9 +89,50 @@ p a:hover {
   color: #ffffff;
 	text-decoration:none;
 }
-/* In this section we will sort our home page styles */
 
-/* Home header */
+/* Footer */
```

Comment:
Good use of footer changes, would recommend fixing css padding issue and make the menu vertical